tools/ExamplesGenerator.ps1

```diff
@@ -158,29 +154,85 @@ function Get-ExternalDocsUrl {
                                 $PathRebuild += $PathSplit[$i] + "/" 
                             }
                             $RebuiltPath = $PathRebuild + "microsoft.graph." + $PathToAppend
-                            $externalDocUrl = $path[$RebuiltPath].get.externalDocs.url
+                            $ExternalDocUrl = $path[$RebuiltPath].get.externalDocs.url
+                            if ([string]::IsNullOrEmpty($ExternalDocUrl)) {
+                                $UriPath2 = $null
+                                if($GraphProfile -eq "beta"){
+                                    $UriPath2 = $BetaCommandListVariantList[$Command]
+                                }else{
+                                    $UriPath2 = $V1CommandListVariantList[$Command]
+                                } 
+                                if(-not([string]::IsNullOrEmpty($UriPath2))){
+                                    $ExternalDocUrl = $Path[$UriPath2].get.externalDocs.url
+                                }
+                                  
+                            }
                         }
                     }
                     if ($MethodName -eq "POST") {
-                        $externalDocUrl = $path[$UriPath].post.externalDocs.url 
+                        $ExternalDocUrl = $Path[$UriPath].post.externalDocs.url
+                        if ([string]::IsNullOrEmpty($ExternalDocUrl)) {
+                            $UriPath3 = $null
```

Comment:
Variable name should be renamed to include the method name:
```suggestion
                            $PostPath = $null
```
The same applies to variables `$UriPath4`...`$UriPath6`.